3rdparty/filament/filament_download.cmake

```diff
@@ -26,9 +26,19 @@ else()
             string(APPEND lib_dir /x86_64/md)
         endif()
     elseif(APPLE)
-        set(FILAMENT_URL https://github.com/google/filament/releases/download/v1.9.19/filament-v1.9.19-mac.tgz)
-        set(FILAMENT_SHA256 2765d0ce60647fc17d1880c4618cf7d6b5343d8be4dad87978c3917d9c723b4e)
-        string(APPEND lib_dir /x86_64)
+        if (APPLE_AARCH64)
+            ## TODO: Update url
+            set(FILAMENT_URL https://github.com/sitic/filament/releases/download/filament/filament-v1.9.19-macos_arm64.tgz)
+            # set(FILAMENT_URL https://github.com/isl-org/open3d_downloads/releases/download/filament/filament-v1.9.19-macos_arm64.tgz)
```

Comment:
arm64 filament binaries are now uploaded to this URL. OK to update URL.
https://github.com/isl-org/open3d_downloads/releases/download/filament/filament-v1.9.19-macos_arm64.tgz